src/main/resources/com/hcl/appscan/jenkins/plugin/Messages.properties

```diff
@@ -22,6 +22,14 @@ error.threshold.invalid=Invalid entry. Enter an integer value.
 error.malformed.url=Malformed URL {0}.
 error.server.unavailable=AppScan plugin encountered an unexpected error while connecting to AppScan on Cloud.
 error.mobile.analyzer=Incorrect user input. The legacy Mobile Analyzer technology is no longer supported, we recommend using our Static Analyzer scanning for Mobile Applications.
+error.dynamic.AppScan360= Dynamic Analyzer is available for AppScan on Cloud only.
+error.dynamic.analyzer.AppScan360= HCL AppScan task failed: Incorrect user input. Dynamic analysis (DAST) is available in AppScan on Cloud only.
```

Comment:
Can you remove "Incorrect user input" from this message? I think the message "HCL AppScan task failed: Dynamic analysis (DAST) is available in AppScan on Cloud only." should suffice.